metadata-io/src/test/java/com/linkedin/metadata/search/LineageSearchServiceTest.java

```diff
@@ -63,13 +64,13 @@ public class LineageSearchServiceTest {
   private static final String ENTITY_NAME = "testEntity";
   private static final Urn TEST_URN = TestEntityUtil.getTestEntityUrn();
 
-  @BeforeTest
+  @BeforeClass
```

Comment:
Wow - I am surprised to see we were previously doing this. No longer these tests felt so painful